OpenDreamRuntime/Procs/DreamProcArguments.cs

```diff
@@ -34,5 +36,35 @@ namespace OpenDreamRuntime.Procs {
             }
             return DreamValue.Null;
         }
+
+        public bool TryGetNamedArgument(string argumentName, [NotNullWhen(true)] out DreamValue? value) {
+            if(NamedArguments == null) {
+                value = DreamValue.Null;
+                return false;
+            }
+
+            if(NamedArguments.TryGetValue(argumentName, out DreamValue lookup)) {
+                value = lookup;
+                return true;
+            }
+
+            value = DreamValue.Null;
+            return false;
```

Comment:
This can be shortened to one line
```suggestion
            return NamedArguments.TryGetValue(argumentName, out value);
```